README.md

```diff
@@ -387,6 +388,39 @@ plugins: [
 ]
 ```
 
+### Base Tag
+
+When the `base` option is used,
+html-webpack-plugin will inject a [base tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base).
+By default, a base tag will not be injected.
+
+The following two are identical and will both insert `<base href="http://example.com/some/page.html">`:
+
+```js
+new HtmlWebpackPlugin({
+  'base': 'http://example.com/some/page.html'
+})
+```
+
+```js
+new HtmlWebpackPlugin({
+  'base': { 'href': 'http://example.com/some/page.html' }
+})
+```
+
+The `target` can be specified with the corresponding key:
+
+```js
+new HtmlWebpackPlugin({
+  'base': {
+    'href': 'http://example.com/some/page.html',
+    'target': '_blank'
```

Comment:
Would this example be vulnerable to target blank hack attempts?

Reply:
In the worst case, copy-pasting this example would only end in a few broken links. IANA [reserves example.com](https://www.iana.org/domains/reserved) for documentation. 

I'd trust IANA with a domain more than I'd trust myself, and if `example.com` were to ever act maliciously, `target="_blank"` would be near the bottom on a list of concerns.

-----

Also, I just ran through the most commonly described phishing scenario on firefox and chrome; both prevent setting `window.opener.location` when opened in a `_blank` context defined by `base`:

on firefox: 

```js
window.opener.location = 'https://phishing-origin.com'
// NS_ERROR_DOM_PROP_ACCESS_DENIED: Access to property denied
```

and on chrome: 

```js
window.opener.location = 'http://phishing-origin.com'
// Unsafe JavaScript attempt to initiate navigation for frame with origin
// 'https://www.original-safe-origin.com' from frame with URL 'https://www.compromised-origin.com/'. 
// The frame attempting navigation is neither same-origin with the target, nor is it the target's parent or opener.
```

Reply:
Okay looks like it is not vulnerable 👍 so that should be no concern